src/api.ts

```diff
@@ -44,6 +44,7 @@ export default class Chromeless<T extends any> implements Promise<T> {
         closeTab: true,
         ...options.cdp,
       },
+      chromeFlags: ['--headless'],
```

Comment:
`remoteDebuggingPort` is already behind `cdp` options and chrome is using port provided there so no need to duplicate that logic here. Whatever you guys want, I am open to changes